src/cmd-cloud-prune

```diff
@@ -414,5 +446,66 @@ def prune_build(s3_client, bucket, prefix, build_id, dry_run):
                 raise Exception(f"Error pruning {build_id}: {e.response['Error']['Message']}")
 
 
+def get_container_tags(meta_json, exclude):
+    container_tags = {}
+    base_oscontainer = meta_json.get("base-oscontainer")
+    if base_oscontainer:
+        tags = base_oscontainer.get("tags", [])
+        # Exclude any tags requested by the caller to be excluded
+        filtered_tags = [tag for tag in tags if tag not in exclude]
+        if filtered_tags:
+            container_tags["tags"] = filtered_tags
+            container_tags["repository_url"] = base_oscontainer.get("image")
+    return container_tags
```

Comment:
OPTIONAL:

hmm. I see you canged container_tags to a dict and are stuffing repository_url in there..

maybe instead let's just return two things:

```suggestion
        if filtered_tags:
            container_repo = base_oscontainer.get("image")
            container_tags = filtered_tags
            return container_repo, container_tags
    return "", []
```

Reply:
fixed in new upload